libs/blocks/marquee/marquee.css

```diff
@@ -189,10 +216,23 @@
   margin-bottom: var(--spacing-s);
 }
 
+.marquee.split a:not(.con-button) {
+  color: var(--color-accent);
+  text-decoration: underline;
+}
+
+.marquee.split .button-justified-mobile {
+  width: 100%;
```

Comment:
Just wanted to check if this should be updated at the typography.css level instead? Is this needed?

Reply:
Good point, adjusted my styles so they adjust flexbox rules accordingly